docs/csharp/language-reference/builtin-types/integral-numeric-types.md

```diff
@@ -0,0 +1,123 @@
+
+## Types to an integer literal
+
+If an integer literal has no suffix, its type is the first of the following types in which its value can be represented:
+
+1. `int`
+1. `uint`
+1. `long`
+1. `ulong`
+
+You can force an integral literal to a type with a smaller range than the default using either an assignment or a cast:
+
+```csharp
+byte byteVariable = 42; // type is byte
+var signedByte = (sbyte)42; // type is sbyte.
+```
+
+You can force an integral literal to a type with a larger range than the default using either assignment, a cast, or a suffix on the literal:
+
+```csharp
+var unsignedLong = 42UL;
+var longVariable = 42L;
+ulong anotherUnsignedLong = 42;
+var anotherLong = (long)42;
+```
+
+The `l` or `L` suffix specifies that the integer literal should a `long`. The `ul` or `UL` suffix specifies a `ulong` type. If the `L` suffix is used on a literal that is larger than 9,223,372,036,854,775,807 (the maximum value of a `long`), the value is a `ulong`.
+
+> [!NOTE]
+> You can use the lowercase letter "l" as a suffix. However, this generates a compiler warning because the letter "l" is easily confused with the digit "1." Use "L" for clarity.
+
+## Conversions
+
+There is an implicit conversion (called a *widening conversion*) between any two integral types where the destination type can store all values of the source type. For example, there is an implicit conversion from `int` to `long` because the range of `int` values is a proper subset of `long`. There is no implicit conversion from any signed integral type to any unsigned integral type. There are implicit conversions from a smaller unsigned integral type to a larger signed integral type. There is also an implicit conversion from any integral type to any floating point type.
+
+You must use an explict cast to convert one integral type to another integral type when an implicit conversion is not defined from the source type to the destination type.
+
+If the value represented by an integer literal exceeds <xref:System.UInt64.MaxValue?displayProperty=nameWithType>, a compiler error [CS1021](../../misc/cs1021.md) occurs.
+
+Use the <xref:System.Numerics.BigInteger?displayProperty=nameWithType> structure to represent an arbitrarily large signed integer.
+
+OTHER TASKS:
+
+1. incorporate implicit and explicit conversion tables.
+1. Consider related "table" consolidation.
+
+## See also
+
+- [C# Language Specification - Integral types](~/_csharplang/spec/types.md#integral-types)
+- [C# Reference](../index.md)
+- [Floating-point types table](../keywords/floating-point-types-table.md)
+- [Default values table](../keywords/default-values-table.md)
+- [Formatting numeric results table](../keywords/formatting-numeric-results-table.md)
+- [Built-in types table](../keywords/built-in-types-table.md)
+- [C# Programming Guide](../../programming-guide/index.md)
```

Comment:
Recently we've removed the links to the programming guide from the operators articles.

```suggestion
```